CMakeLists.txt

```diff
@@ -602,6 +602,34 @@ MACRO(ADD_GEMRB_PLUGIN plugin_name)
 	ENDIF (APPLE)
 ENDMACRO(ADD_GEMRB_PLUGIN)
 
+# TODO: think of a way to add additional libs for linking
+MACRO(ADD_GEMRB_PLUGIN_TEST plugin_name)
+	IF (BUILD_TESTING)
+		ADD_EXECUTABLE(Test_${plugin_name} ${ARGN})
+
+		TARGET_LINK_LIBRARIES(Test_${plugin_name} GTest::gtest GTest::gtest_main gemrb_core)
+
+		# copy test executables to where the core lib is
```

Comment:
I don't understand why we do this. seems like we are making/have mad more work for ourselves by not being able to just use the build dir. I've never had to manually juggle things in other cmake+ctest projects. Is it because we aren't using cpack?

Reply:
I've never used CPack, so I don't know.

It's just that plugins (and their tests now) end up in subdirectories where there is no `gemrb_core.dll` in Windows' lookup paths.

Reply:
For future reference, this is the layout of MSVC. 3rd party DLLs are not where the tests need them.
![msvc](https://github.com/gemrb/gemrb/assets/238558/12210843-1749-48a8-a7d6-d5a86c0364fa)

Reply:
Great, now the plugin tests are 100% wrong here but at least no more library loading failures. I'll leave it this way for now until we have some better approach to cover all uses and build systems.

Reply:
At least disable these tests on windows then, otherwise we won't get any successful builds.

Reply:
I'm checking this since I want to rule out that there is actually something wrong now.

Reply:
Ok. I remember before we ironed out vcpkg people used to have problems building/running with VS for a similar reason, it couldn't find the dlls. If you can make it work with the dlls near the targets, I see no problem with changing that aspect. They're all dumped together when packaging anyway.